app/views/events/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row">
+  <div class="col-md-6 form-new-user col-md-offset-3">
+    
+    <%= form_for(@event) do |f| %>
+      <%= f.label :description %><br>
+      <%= f.text_area :description, class: "form-control" %><br>
+    
+      <%= f.submit 'Create', class: 'btn btn-primary' %>
+    <% end %>
```

Comment:
- [ ] For new event form, we need more than just a description input for the event. We need also title, location and date (which is crucial in deciding whether the event is upcoming or past).

Reply:
I really fixed errors by I forget to push changes to develop branch. :)